depthai_ros_driver/CMakeLists.txt

```diff
@@ -109,14 +111,14 @@ target_link_libraries(gen2_ros_queue ${catkin_LIBRARIES} depthai range-v3)
 add_executable(gen2_ros_driver src/depthai_loader.cpp)
 target_link_libraries(gen2_ros_driver gen2_ros_queue gen2_ros_pipeline_loader gen2_ros_conversion)
 
-add_library(img_detections_stamper src/img_detections_stamper.cpp)
-target_link_libraries(img_detections_stamper ${catkin_LIBRARIES})
+add_library(common_stampers src/common_stampers.cpp)
+target_link_libraries(common_stampers ${catkin_LIBRARIES} depthai-core)
 
-add_library(img_detections_stamper_nodelet src/img_detections_stamper_nodelet.cpp)
-target_link_libraries(img_detections_stamper_nodelet img_detections_stamper)
+add_library(common_stamper_nodelets src/common_stamper_nodelets.cpp)
+target_link_libraries(common_stamper_nodelets common_stampers)
 
 add_executable(img_detections_stamper_node src/img_detections_stamper_node.cpp)
-target_link_libraries(img_detections_stamper_node img_detections_stamper)
+target_link_libraries(img_detections_stamper_node common_stampers)
```

Comment:
don't we also need ROS1 nn_stamper_node as executable?

Reply:
depthai_ros_driver/src/img_detections_stamper_node.cpp is still there. Could you expand on this please?

Reply:
In common stampers.hpp, we have 
`using ImgDetectionsStamperNode = ImgDetectionsStamper<ROS1Node<>>;` and
`using NNStamperNode = NNStamper<ROS1Node<>>;`

For ros1 node, we also need a main function that has
```
int main(int argc, char** argv) {
    rr::NNStamperNode node;
    node.init();
```
I'm talking about that cpp file